File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/op/atts/service/schema/annotation/AttName.java
package ru.citeck.ecos.records3.record.op.atts.service.schema.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

@Retention(RetentionPolicy.RUNTIME)
@Target({ElementType.FIELD, ElementType.METHOD})
public @interface AttName {

    String value() default "";
}

File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/op/atts/service/schema/annotation/AttName.kt
package ru.citeck.ecos.records3.record.op.atts.service.schema.annotation

@Retention(AnnotationRetention.RUNTIME)
@Target(
    AnnotationTarget.FIELD,
    AnnotationTarget.FUNCTION,
    AnnotationTarget.PROPERTY_GETTER,
    AnnotationTarget.PROPERTY_SETTER
)
annotation class AttName(val value: String = "")
